### PR TITLE
Replaced social icon images with font-awesome equivalents

### PR DESCRIPTION
**Pull request by xtbe**
Social icon images replaced with font-awesome equivalents
Updated styling of font-awesome icons to align with website look 'n feel
Changed social icon spacing so that all icons are on one row after adding Twitch icon

##### Checklist

- [x] `npm test` passes
- [x] documentation is changed or added
- [x] contribution guidelines followed [here](https://github.com/AdoptOpenJDK/openjdk-website/blob/master/CONTRIBUTING.md)

Tested on following browsers and devices:

- [x] PC - Mac - Safari
- [x] PC - Windows - Chrome
- [x] PC - Windows - Firefox
- [x] PC - Windows - Internet Explorer
- [x] PC - Windows - Edge
- [x] iPad - Mac - Safari
- [x] IPhone - Mac - Safari

### CODE REVIEW

**Comment by gdams** [comment]
Can you slightly shrink the size of the icons? They seem a bit overwhelming now

**Comment by xtbe** [comment]
> Can you slightly shrink the size of the icons? They seem a bit overwhelming now

Ok, will have a look to shrink them a little bit

**Comment by xtbe** [comment]
See sample below, just indicate with which size we should go

image

icon 1 = 1.00em
icon 2 = 1.05em
icon 3 = 1.10em
icon 4 = 1.15em
icon 5 = 1.33em

After changing the size I will adapt the spacing as well as we will have some additional spacing because of decreasing the size.

File: src/scss/styles-1-large-main.scss [hunk 1]
```diff
@@ -147,6 +147,26 @@ $headersize: 1.4rem;
...
+}
+
+.fa-stack {
+  border-radius: 50px;
+  box-sizing: border-box;
+  border: 1px solid transparent;
+  transition:.5s;
+  background:#ffffff;
+  opacity: .5;
+
```
Comment (karianna) [inline src/scss/styles-1-large-main.scss]: whitespace

Comment (xtbe) [reply src/scss/styles-1-large-main.scss]: Ok, Wasn't sure they had to be removed, will take them out as well when changing the icon size

**Comment by gdams** [comment]
@xtbe let's go with `icon 3 = 1.10em`?

**Comment by xtbe** [comment]
> @xtbe let's go with `icon 3 = 1.10em`?

Ok,  will use that one, we can evaluate when changed

File: src/scss/styles-1-large-main.scss (hunk 1 above)
Comment (xtbe) [reply src/scss/styles-1-large-main.scss]: Checked spacing, obsolete images removed

**Pull request merged**